packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -176,9 +176,15 @@ Future<XcodeBuildResult> buildXcodeProject({
   }
 
   Map<String, String> autoSigningConfigs;
+
+  final Map<String, String> buildSettings = await app.project.buildSettingsForBuildInfo(
```

Comment:
can this be defined after line 185, so we don't need to await this if we never hit the if block?

Reply:
Before, `buildSettings` was set in the `showBuildSettingsCommand` code I deleted.  Now it's calculated earlier but still used on line 380.